src/arch/x86/X86CPU.py

```diff
@@ -80,7 +80,6 @@ class DefaultX86FUPool(FUPool):
         PredALU(),
         WritePort(),
         RdWrPort(),
-        IprPort(),
```

Comment:
I think this should be replaced with `System_Unit`

Reply:
Done